### DISCUSSION TITLE
Hammer Q.Vadis (7.0)

### DISCUSSION THREAD

**Discussion by mistersoftee** [cagnulein/qdomyos-zwift#1620] [General]
Hi Roberto,

I have a Hammer Q.Vadis treadmill with bluetooth and Zwift integration. Unfortunately it's has no auto-incline or speed. I tried to use it with QZ and the tiles are showing up as soon as I power on the treadmill, but it doesn't show metrics. If I press the start button, the treadmill starts. But the speed and cadence functions doing weird things. And if I change settings on the treadmill, the tiles stay at 0. 

Is there anything I can do in the options to get it running?

Best 
Stefan

**Comment by cagnulein** [comment]
hi, i'm in vacation but in the mean while provide me a debug log
https://github.com/cagnulein/qdomyos-zwift/wiki/How-do-i-get-the-debug-log-in-case-something-doesn't-work%3F

i will be back next week

Il giorno ven 25 ago 2023 alle 13:02 mistersoftee ***@***.***>
ha scritto:
